assets/src/edit-story/components/helpCenter/popup.js

```diff
@@ -55,13 +55,19 @@ const Controller = styled.div`
   ${({ state }) => transitionStyles[state]}
 `;
 
-export function Popup({ isOpen, popupId, children, ariaLabel }) {
+export function Popup({
+  isOpen,
+  popupId,
+  children,
+  ariaLabel,
+  shouldKeepMounted,
+}) {
   return (
     <ScheduledTransition
       in={isOpen}
       timeout={DURATION}
-      mountOnEnter
-      unmountOnExit
+      mountOnEnter={!shouldKeepMounted}
+      unmountOnExit={!shouldKeepMounted}
```

Comment:
Using the help center structure as the default here when `shouldKeepMounted` is **not** passed in the popup will not mount until isOpen is true and then it will unmount when it is closed. If we need access to the contents of the popup without it being opened (as is the case for the checklist) then we want to make both of these false.